### ISSUE TITLE
Unable to set clipboard via xclip

### ISSUE THREAD

**Issue by chrisjunlee** [Morantron/tmux-fingers#8]
**Expected Behavior:**
According to the wiki 'set -g @fingers-copy-command 'xclip -selection clipboard' will allow tmux-fingers to copy content into the xselection clipboard.

**Actual Behavior:**
Selection does not appear in host clipboard.

**Root Cause**
Clipboard is cleared due to xclip being killed.

Narrowed it down to this line: 
```bash
fingers.sh:copy_result():
49  echo -n "$result" | $FINGERS_COPY_COMMAND
```
At that point, xclip exits immediately, leaving the clipboard ownerless. Apparently the clipboard needs a 'owner' at all times in order to persist.

Have also tried setting the command with nohup to no avail: 
```bash
.tmux.conf:
    set -g @fingers-copy-command 'nohup xclip -selection clipboard'
```

Not sure if we can make this work through configuring *@fingers-copy-command* and executing within the plugin. This may be a plugin manager issue.

Interesting to note that xclip persists when called outside the plugin context. For instance:
```bash
.tmux.conf:
    bind b run-shell 'echo test | nohup xclip -selection clipboard > /dev/null'
```
Will set the clipboard correctly.

**Comment by Morantron** [comment]
Thanks for reporting and thorough description! I'll need to investigate further what's happening, I'll probably change the implementation to `tmux copy-pipe` which seems to cover the issue.

Found [an article](https://wiki.archlinux.org/index.php/tmux) in Arch Linux wiki commenting similar to what you mentioned.

But anyway, there seems to be an error in the way the result is piped.

**Comment by Morantron** [comment]
Fixed in `0.1.3` :dancers:. Looks like using evil `eval` makes it work here, not sure why TBH :trollface:
